database.hh

```diff
@@ -827,6 +827,10 @@ public:
     future<> snapshot(sstring name);
     future<std::unordered_map<sstring, snapshot_details>> get_snapshot_details();
 
+    /*!
+     * \brief write the table schema to a schema file at the given directory
```

Comment:
Would have been nice to give more context in this comment... That this function has to do with snapshots (this "schema file" isn't something we use normally in normal database operation), and the "schema file" is really a CQL command to recreate the schema.
I think it would also be better to rename this function **write_snapshot_schema** or similar, to make clearer this function is just used by snapshots (if "snapshot" is the right word... Or is it "backup"?).

Reply:
ok, I'll take your view, though it's a valid schema, if anyone would need the schema description (i.e pretty print instead of the redirection operator) it would be useful.

Reply:
So maybe write_schema_as_cql() or something like that?
To emphasize that it writes CQL, it's not the normal function we use to persist a schema to disk or something.

Reply:
I like that better